bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -254,33 +254,33 @@ spec:
                exit 0
           fi
           echo "Adding ICMP drop rule for '$3' "
-          if iptables -C CHECK_ICMP_SOURCE -p icmp -s $3 -j ICMP_ACTION
+          if iptables -C AZURE_CHECK_ICMP_SOURCE -p icmp -s $3 -j AZURE_ICMP_ACTION
           then
                echo "iptables already set for $3"
           else
-               iptables -A CHECK_ICMP_SOURCE -p icmp -s $3 -j ICMP_ACTION
+               iptables -A AZURE_CHECK_ICMP_SOURCE -p icmp -s $3 -j AZURE_ICMP_ACTION
           fi
           EOF
 
           echo "I$(date "+%m%d %H:%M:%S.%N") - drop-icmp - start drop-icmp ${K8S_NODE}"
-          iptables -X CHECK_ICMP_SOURCE || true
-          iptables -N CHECK_ICMP_SOURCE || true
-          iptables -F CHECK_ICMP_SOURCE
-          iptables -D INPUT -p icmp --icmp-type fragmentation-needed -j CHECK_ICMP_SOURCE || true
-          iptables -I INPUT -p icmp --icmp-type fragmentation-needed -j CHECK_ICMP_SOURCE
-          iptables -N ICMP_ACTION || true
-          iptables -F ICMP_ACTION
-          iptables -A ICMP_ACTION -j LOG
-          iptables -A ICMP_ACTION -j DROP
-          oc observe pods -n openshift-sdn -l app=sdn -a '{ .status.hostIP }' -- /var/run/add_iptables.sh
+          iptables -X AZURE_CHECK_ICMP_SOURCE || true
+          iptables -N AZURE_CHECK_ICMP_SOURCE || true
+          iptables -F AZURE_CHECK_ICMP_SOURCE
+          iptables -D INPUT -p icmp --icmp-type fragmentation-needed -j AZURE_CHECK_ICMP_SOURCE || true
+          iptables -I INPUT -p icmp --icmp-type fragmentation-needed -j AZURE_CHECK_ICMP_SOURCE
+          iptables -N AZURE_ICMP_ACTION || true
+          iptables -F AZURE_ICMP_ACTION
+          iptables -A AZURE_ICMP_ACTION -j LOG
+          iptables -A AZURE_ICMP_ACTION -j DROP
+          /host/usr/bin/oc observe pods -n openshift-sdn -l app=sdn -a '{ .status.hostIP }' -- /var/run/add_iptables.sh
```

Comment:
This seems not safe; is `oc` statically linked, and is it guaranteed to _always_ be statically linked? I think you have to do `chroot /host oc ...` though of course then you also have to write the script out to `/home/var/run` rather than `/var/run`

Reply:
I think we've glossed over this so far but due to FIPS needing openssl for Go, we can't rely on these things being fully static.

Reply:
The other pattern is to use e.g. `systemd-run -qPG -- oc observe pods -n openshift-sdn -l app=sdn -a '{ .status.hostIP }' -- /var/run/add_iptables.sh`.

Reply:
@mjudeikis fyi

Reply:
The commit message already describes: 

1. Change to use mountPath: /host
2. Change iptables chain names to not conflict with existing workaround(s)

For SDN, I'll probably need to change upstream to use systemd-run (or whatever ends up being used) and back port.

The OVN version of this fix will start with whatever consensus decides is best.